tests/client-middleware-cache.js

```diff
@@ -117,37 +123,4 @@ describe("Client Middleware", () => {
       });
     });
   });
-
```

Comment:
Good idea with the support dir.
Could you possibly share the failing tests error?